docs/src/pages/[platform]/components/link/react.mdx

```diff
@@ -47,19 +47,45 @@ To create a Link which opens in a new tab, use the `isExternal` prop. Under the
 
 ### Routing Libraries
 
-You can use a Link with any React routing library that supports custom components. Below is an example using Link with React Router v5, in which the Link is passed to the `component` prop as a custom navigation component:
+You can use a Link with any React routing library that supports custom components. Below are examples for both React Router v5 and v6:
+
+#### React Router v5
```

Comment:
See my comment below, that code works in react-router v5 too, so we don't need different examples for v5/6. 

v5: https://codesandbox.io/s/crazy-drake-ny7qz8?file=/src/App.tsx